File: GTP2/ej4.cpp
#include <iostream>

using namespace std;

int aleatorio() {
    return rand()%100;
}

void rellenar(int a[15]) {
    for (int i = 0; i < 15; i++) {
        a[i] = aleatorio();
    }
}

int main() {
    srand(time(NULL));
    
    int a[15];
    rellenar(a);

    for (int *i = &a[0]; i < &a[14]; i++) {
        cout << *i << "\t(" << i << ")" << endl;
    }

    return 0;
}

File: GTP2/ej4.go
package main

import (
	"fmt"
	"math/rand"
)

func main() {
	// new permite alojar memoria para un tipo
	var arr = new([15]int)

	rellenar(arr)

	for _, i := range arr {
		fmt.Println(i)
	}
}

func rellenar(arr *[15]int) {
	for i := range arr {
		arr[i] = rand.Intn(101)
	}
}
